arch/Config.pl

```diff
@@ -797,6 +799,121 @@
 
   $_ =~ s/CONFIGURE_DEP_LIB_PATH/$sw_dep_lib_path/g ;
 
+    $_ =~ s/CONFIGURE_PERL_PATH/$sw_perl_path/g ;
```

Comment:
@DWesl 
Daniel,
Why do you replicate these 100+ lines of perl? This code already exist from about line 440 through 560.

Reply:
Processing preamble.  I didn't know what `LIB_BUNDLED` and `LIB_EXTERNAL` assumed to be defined, so I just moved everything there.
I'll try paring this down sometime tomorrow.  It will likely be next week before I can test `dmpar` configurations, but I might be able to get to `serial` and `smpar` builds before then.  I don't remember what I tested the first time through.

Reply:
Cleaned to just the variables referenced in `LIB_EXTERNAL` and `LIB_BUNDLED`